backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Controllers/UserController.cs

```diff
@@ -34,13 +35,20 @@ public async Task<ActionResult<UserDto>> Post([FromBody] UserRegisterDto user)
             return Ok(createdUser);
         }
 
-
         [HttpPut("{userId}")]
         public ActionResult<UserDto> Update(int userId, UserDto userDto)
         {
             return Ok(userDto);
         }
 
+        [HttpPost]
```

Comment:
Please, don't create endpoint with same url even if they have different parameters
You can name this one /users/tags